### ISSUE TITLE
Internet Connection Toast

### ISSUE THREAD

**Issue by suntwalriya** [sanscript-tech/medical-emergency-flutter-node#5]
- Toast if the Internet Connection is working or not

**Comment by ShruthiShriT** [comment]
can you assign me this ?

**Comment by 7tena** [comment]
Can you assign me this?

**Comment by 7tena** [comment]
Can I work on this?

**Comment by suntwalriya** [comment]
@7tena any updates?

**Comment by 7tena** [comment]
Yes. I've been busy with exams this week but I'm working on it. I will try
to get it done as soon as possible. Most probably will finish it by
tomorrow night.

On Fri, 30 Oct 2020,

**Comment by 7tena** [comment]
Yes. Will send a PR by tomorrow.

**Comment by samflab** [comment]
@7tena what's the status? It's been a week now
